bin/ember-template-lint.js

```diff
@@ -131,7 +131,9 @@ async function run() {
   }
 
   if (options.compactTodo) {
-    let { compacted } = compactTodoStorageFile(options.workingDirectory);
+    let { compacted } = compactTodoStorageFile(options.workingDirectory, {
+      engine: 'ember-template-lint',
+    });
```

Comment:
We were just chatting about this, I think we should consider making `compactTodoStorageFile` work across all engines. As it stands, I worry that we might drop unrelated todos from other engines during compaction...

Might need another test for this over in the todo utils...

Reply:
Done and done.

Reply:
Hmm, I would have thought that you don't need the second arg anymore?